test/src/main/java/org/apache/accumulo/test/DumpConfigIT.java

```diff
@@ -58,20 +58,22 @@ public void configure(MiniAccumuloConfigImpl cfg, Configuration hadoopCoreSite)
       justification = "user.dir is suitable test input")
   @Test
   public void test() throws Exception {
-    File folder = tempDir.toPath().resolve(testName() + "/").toFile();
-    assertTrue(folder.isDirectory() || folder.mkdir(), "failed to create dir: " + folder);
-    File siteFileBackup = folder.toPath().resolve("accumulo.properties.bak").toFile();
-    assertFalse(siteFileBackup.exists());
-    assertEquals(0, exec(Admin.class, "dumpConfig", "-a", "-d", folder.getPath()).waitFor());
-    assertTrue(siteFileBackup.exists());
-    String site = FunctionalTestUtils.readAll(Files.newInputStream(siteFileBackup.toPath()));
+    Path folder = tempDir.resolve(testName());
+    if (!Files.isDirectory(folder)) {
+      Files.createDirectories(folder);
+    }
```

Comment:
could add similar assertion: `assertTrue(Files.isDirectory(dir))` or move to util method I mentioned earlier